blog/_src/posts/2017-06-06-spring-2017-pl-junior-retrospective.md

```diff
@@ -0,0 +1,162 @@
+    Title: Spring 2017 PL Junior Retrospective
+    Date: 2017-06-06T11:38:25
+    Tags: DRAFT, PL Junior, by Ben Chung, by Milo Davis, by Ming-Ho Yee, by Matt Kolosick, by Dustin Jamner, by Artem Pelenitsyn, by Julia Belyakova, by Sam Caldwell
+
+The [PL Junior Seminar][pljunior] is for beginning PhD and interested
+undergrad and masters students to understand the foundations of
+programming languages research. It serves to fill in background
+knowledge and get up to speed with different areas of PL research.
+
+For the spring 2017 instance of PL Junior we chose program synthesis,
+the sequent calculus, and logic programming as topics we wanted to
+learn more about. We also did two group paper readings for Luca
+Cardelli's [Typeful Programming][typeful] and Alan Kay's
+[Early History of Smalltalk][smalltalk]. At the same time, we changed
+up the format from the previous semester.
+
+<!-- more -->
+
+
+## Format
+
+As discussed in [last fall's retrospective][fall16retro], we wanted to
+move from group reading and discussion towards weekly presentations.
+Reading a paper or some other material in order to understand it well
+enough to present is quite a different experience compared to the
```

Comment:
Maybe change to "reading a paper to prepare a presentation"? Currently sounds a bit awkward to me.